src/Grpc.Net.Client/GrpcChannelOptions.cs

```diff
@@ -65,6 +66,56 @@ public sealed class GrpcChannelOptions
         /// </summary>
         public int? MaxReceiveMessageSize { get; set; }
 
+        /// <summary>
+        /// Gets or sets the maximum retry attempts. This value limits any retry and hedging attempt values specified in
+        /// the service config.
+        /// <para>
+        /// Setting this value alone doesn't enable retries. Retries are enabled in the service config, which can be done
+        /// using <see cref="ServiceConfig"/>.
+        /// </para>
+        /// <para>
+        /// A <c>null</c> value removes the maximum retry attempts limit. Defaults to 5.
+        /// </para>
+        /// <para>
+        /// Note: Experimental API that can change or be removed without any prior notice.
+        /// </para>
+        /// </summary>
+        public int? MaxRetryAttempts { get; set; }
```

Comment:
I agree somewhat. I choose this name because it matches what Java has on its channel.

Also retries and hedging are both retry strategies. One strategy happens serially (retries) and the other in parallel (hedging).

I prefer `MaxRetryAttempts` over `MaxAttempts` or `MaxRetryOrHedgingAttempts`.